packages/test-cases/cases/wrappers/wasm-rs/json-type/src/lib.rs

```diff
@@ -3,10 +3,26 @@ pub use wrap::*;
 use polywrap_wasm_rs::JSON;
 use serde_json::*;
 
-pub fn from_json(input: InputFromJson) -> Pair {
```

Comment:
I propose keeping the from_json and to_json tests for the Rust JSON implementation. They should still work even if we've removed the analogous capability from the AS code.